airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -113,9 +113,21 @@ function main() {
     . /airbyte/sshtunneling.sh
     openssh "${PROJECT_DIR}/ssh.json"
     trap 'closessh' EXIT
+
+    # We don't run dbt 1.0.x on all destinations (because their plugins don't support it yet)
+    # So we need to only pass `--event-buffer-size` if we have DBT 1.0.0
+    # For some reason, `dbt --version` outputs to stderr, so we need to redirect it to stdout
+    dbt --version 2>&1 | grep -q 'installed version: 1.0.0'
```

Comment:
What if we upgrade dbt to `42.2.1`?

WHy not just try to run a dummy dbt cli with `--event-buffer-size` and see if that passes?

Reply:
> 42.2.1

I'm not sure what that's referring to, isn't 1.0.x the highest version?

> run a dummy dbt cli

I feel like this would take more setup for basically the same outcome? E.g. if it just runs `dbt --event-buffer-size=10000 run --project-dir "/nonexistent/path"`, the script would still need to grep the output (to check whether it failed due to not recognizing `--event-buffer-size`, or because it just couldn't find the project file)

Unless you're suggesting to try actually running the real command with `--event-buffer-size`, and if it fails then rerun without it?

Reply:
> I'm not sure what that's referring to, isn't 1.0.x the highest version?

Yes today it's the highest, I was just saying when in the future we upgrade dbt again, and the string "1.0.0" being looked for won't be in the output anymore

> I feel like this would take more setup for basically the same outcome? 

you can do:
![Screenshot 2022-03-24 at 16 49 37](https://user-images.githubusercontent.com/7718175/159956437-11d3d743-6d2c-4799-93db-ba20797dd7d8.png)

Reply:
aaahhh gotcha. yeah this looks a lot nicer, will do!